app/controllers/v0/sessions_controller.rb

```diff
@@ -78,7 +78,8 @@ def handle_completed_slo
 
       errors = []
       errors.concat(logout_response.errors) unless logout_response.validate(true)
-      errors << "Logout Request not found! ID=#{logout_response&.in_response_to}" if logout_request.nil?
+      errors << 'inResponseTo attribute is nil!' if logout_response&.in_response_to.nil?
```

Comment:
If having the `in_response_to` is important for debugging the individual error instances, we can re-add it with `Raven.extra_context(id: logout_response&.in_response_to)` (or `user_context` maybe?) per https://docs.sentry.io/clients/ruby/context/